api-report/container-loader.api.md

```diff
@@ -319,16 +230,6 @@ export class Loader implements IHostLoader {
     readonly services: ILoaderServices;
 }
 
-// @public (undocumented)
-export enum ReconnectMode {
```

Comment:
We should add a note about these interfaces not being exported anymore in breaking.MD.

Reply:
I personally do not care - that is all internal details that nobody should have used. The only way to reach out to DeltaManager instance was by guessing / casting, whoever did it is not supported.